sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java

```diff
@@ -252,12 +279,12 @@ public JsonNode jsonNodeFrom(QueryOuterClass.Value value) {
   protected static final class CounterCodec extends FromProtoValueCodec {
     @Override
     public Object fromProtoValue(QueryOuterClass.Value value) {
-      return String.valueOf(value.getInt());
+      return String.valueOf(Values.int_(value));
```

Comment:
I don't think this is right: remove `String.valueOf()`.

Reply:
`String.valueOf()` was in there originally though, I didn't add that.

Reply:
Comment above:
```
  // NOTE! Should be able to just use `LongCodec` but SGv1 seems to expect JSON String as
  // return value, and not more logical choice of integral number
```
This is the interoperability concern I mentioned in the previous PR. v1 does that for CQL bigint (Java long) too.

Reply:
Oh. My bad -- I somehow didn't see it is for `Counter` and (not sure how) thought it was for numbers.

So never mind. I should have checked the context since I bumped into this myself etc :-D